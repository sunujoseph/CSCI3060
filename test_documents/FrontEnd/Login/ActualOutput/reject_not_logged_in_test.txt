path recieved: C:\Users\pwner\Documents\Uoit\Year 5 Semester 2\CSCI 3060U\Project\CSCI3060\test_documents\FrontEnd\Login\ActualDTF\reject_not_logged_in_test.txt
Error: Not Logged In
Error: Not Logged In
Error: Not Logged In
Error: Not Logged In
Error: Not Logged In
Error: Not Logged In
Error: Not Logged In
Enter Username: 