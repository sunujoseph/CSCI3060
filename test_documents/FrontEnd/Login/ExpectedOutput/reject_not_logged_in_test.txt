Error: Not logged in
Error: Not logged in
Error: Not logged in
Error: Not logged in
Error: Not logged in
Error: Not logged in
Error: Not logged in
Enter Username: 